nucleus/parent/pom.xml

```diff
@@ -879,6 +881,7 @@
                             <java.util.logging.manager>${test.logManager}</java.util.logging.manager>
                             <java.util.logging.config.useDefaults>true</java.util.logging.config.useDefaults>
                             <java.util.logging.config.defaultLevel>${test.logLevel}</java.util.logging.config.defaultLevel>
+                            <glassfish.suspend>${glassfish.suspend}</glassfish.suspend>
```

Comment:
Just wondering where this is for. I mean, I know of course what suspend does, but what does defining is as a property do here over just using ${glassfish.suspend} elsewhere?

Reply:
The surefire plugin runs all tests in a separate JVM. System properties are not passed to this new JVM. So it's necessary to have this to pass the `glassfish.suspend` sys prop to tests.

Reply:
I have been using glassfish.suspend with specifically the surefire plug-in a lot of times. No sure what is different then, but okay.